pandas/tests/frame/test_analytics.py

```diff
@@ -789,10 +812,44 @@ def test_stat_operators_attempt_obj_array(self, method):
             if method in ['sum', 'prod']:
                 tm.assert_series_equal(result, expected)
 
-    def test_mean(self, float_frame_with_na, float_frame, float_string_frame):
-        assert_stat_op_calc('mean', np.mean, float_frame_with_na,
-                            check_dates=True)
-        assert_stat_op_api('mean', float_frame, float_string_frame)
+    @pytest.mark.parametrize('op', ['mean', 'std', 'var',
+                                    'skew', 'kurt', 'sem'])
```

Comment:
I see you decide to add some `pytest` idiom here.  There are a bunch of other places (that you modified) that definitely merit some more refactoring as you probably know...

If you are intending to add more parameterization, then I'll comment as needed.